src/OctoshiftCLI.Tests/gei/Commands/MigrateRepoCommandTests.cs

```diff
@@ -11,14 +11,20 @@ namespace OctoshiftCLI.Tests.GithubEnterpriseImporter.Commands
     public class MigrateRepoCommandTests
     {
         private const string Target_Api_Url = "https://api.github.com";
+        private const string GHES_Api_Url = "https://api.ghes.com";
+        private const string Azure_Connection_String = "DefaultEndpointsProtocol=https;AccountName=myaccount;AccountKey=mykey;EndpointSuffix=core.windows.net";
+        private const string Source_Org = "foo-source-org";
+        private const string Source_Repo = "foo-repo-source";
+        private const string Target_Org = "foo-target-org";
+        private const string Target_Repo = "foo-target-repo";
```

Comment:
nit: for the sake of consistency, can we make all of these private constants all caps? --> i.e `GHES_API_URL`